src/views/bootablevolumes/actions/components/EditBootableVolumesModal.tsx

```diff
@@ -41,51 +41,106 @@ const EditBootableVolumesModal: FC<EditBootableVolumesModalProps> = ({
     () => Object.keys(convertResourceArrayToMap(preferences)).sort((a, b) => a.localeCompare(b)),
     [preferences],
   );
-  const initialMetadata: BootableVolumeMetadata = useMemo(() => {
+
+  const instanceTypesAndSizesObject = useMemo(
+    () => getInstanceTypesWithSizes(instanceTypesNames),
+    [instanceTypesNames],
+  );
+
+  const initialParams = useMemo(() => {
+    const instanceTypeLabel =
```

Comment:
How would you feel about changing the name to `instanceTypeCode`? 'Label' makes me think of the display name.

Reply:
Ah, ok. That makes sense. =)